src/core/services/google-maps-api-wrapper.ts

```diff
@@ -112,6 +112,20 @@ export class GoogleMapsAPIWrapper {
    */
   getNativeMap(): Promise<mapTypes.GoogleMap> { return this._map; }
 
+  getAnimation(name: string): Promise<any> {
```

Comment:
I think we dont need a new method in the maps wrapper. This can be handled by the marker manager.